src/fswAlgorithms/transDetermination/dvAccumulation/dvAccumulation.c

```diff
@@ -105,10 +105,7 @@ void dvAccumulation_QuickSort (AccPktDataMsgPayload *A, int start, int end)
 {
     /*! - Create an auxiliary stack array. This contains indicies. */
     int stack[MAX_ACC_BUF_PKT];
-    if((end-start + 1) > MAX_ACC_BUF_PKT)
-    {
-        BSK_PRINT(MSG_ERROR,"dvAccumulation_QuickSort: Stack insufficiently sized for quick-sort somehow.");
-    }
+    assert(("Stack insufficiently sized for quick-sort.", (end-start + 1) <= MAX_ACC_BUF_PKT));
```

Comment:
Apparently my compiler *does* choke on the comma-based way of adding a message to `assert`, as you predicted. In the interest of reducing compiler warnings, can we use the `&&`-based approach instead here?

Reply:
Thanks for checking. Can you please confirm that what I have added back is what you tested? If so I will mark as resolved.

Reply:
I recompiled your branch and don't see the warning anymore :+1: